### ISSUE TITLE
hint does not query Mesh Identifiers for chemical substances

### ISSUE THREAD

**Issue by mmayers12** [biothings/bte_schema#20]
Mesh IDs for chemicals do not return any results when queried by Hint. However, MyChem has the ability to query for these mesh IDs.

See Example below:

```python
from biothings_explorer.user_query_dispatcher import SingleEdgeQueryDispatcher
from biothings_explorer.hint import Hint
from biothings_explorer.registry import Registry

reg = Registry()
ht = Hint()

mesh_id = 'D020110'
ht.query(mesh_id)
```
```
{'Gene': [],
 'SequenceVariant': [],
 'ChemicalSubstance': [],
 'DiseaseOrPhenotypicFeature': [],
 'Pathway': [],
 'MolecularActivity': [],
 'CellularComponent': [],
 'BiologicalProcess': [],
 'Anatomy': [],
 'PhenotypicFeature': []}
```

```python
from biothings_client import get_client
mc = get_client('chem')

r = mc.query('drugcentral.xrefs.mesh_descriptor_ui:{}'.format(mesh_id))
len(r['hits'])
```

```
1
```

```python
print([k for k in r['hits'][0].keys() if not k.startswith('_')])
print([v.get('id') for k, v in r['hits'][0].items() if not k.startswith('_')])
```

```
['chebi', 'chembl', 'drugbank', 'drugcentral', 'ginas', 'pubchem']
['CHEBI:91706', None, 'DB06762', None, None, None]
```
```python
db_id = r['hits'][0]['drugbank']['id']
ht.query(db_id)
```

```
{'Gene': [],
 'SequenceVariant': [],
 'ChemicalSubstance': [{'chembl': 'CHEMBL1159',
   'drugbank': 'DB06762',
   'name': 'PINACIDIL',
   'pubchem': 4826,
   'umls': 'C0071074',
   'display': 'chembl(CHEMBL1159) drugbank(DB06762) name(PINACIDIL) pubchem(4826) umls(C0071074) ',
   'type': 'ChemicalSubstance',
   'primary': {'identifier': 'chembl',
    'cls': 'ChemicalSubstance',
    'value': 'CHEMBL1159'}}],
 'DiseaseOrPhenotypicFeature': [],
 'Pathway': [],
 'MolecularActivity': [],
 'CellularComponent': [],
 'BiologicalProcess': [],
 'Anatomy': [],
 'PhenotypicFeature': []}
```

**Comment by kevinxin90** [comment]
Thanks for the feedback. If you upgrade the bte_schema package, you should be able to query for mesh chemical ID now.

**Comment by kevinxin90** [comment]
Fixed in: d98957d30538ad1fb83d65e4719aedf588458a51
